### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Abinashbunty**
Updated the sponsors correctly

#### CheckList
- [ ] Did you add an unittest for your new example or defect fix?
- [ ] Did you add documents for your new example?
- [x] All CIs are green? (You can check it after submitting)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -626,13 +627,13 @@ If you or your company would like to support this project, please consider:
...
-## Sponsors
+## <a id="sponsors"></a>Sponsors
 
-### [JetBrains](https://www.jetbrains.com/)
+### <a id="JetBrains"></a>[JetBrains](https://www.jetbrains.com/)
 
 They are providing a free license of their IDEs for this OSS development.   
 
-### [1Password](https://github.com/1Password/1password-teams-open-source)
+### <a id="1password"></a>[1Password](https://github.com/1Password/1password-teams-open-source)
```
Comment (AtsushiSakai) [inline README.md:636]: Why did you add a tag?

Comment (Abinashbunty) [reply README.md:636]: The tag makes a direct reference to the sponsor when clicked on at the sponsors from the list items towards the top in Readme.

Comment (AtsushiSakai) [reply README.md:636]: Is it needed? I think header TOC items already links to each Markdown header.

Comment (Abinashbunty) [reply README.md:636]: Actually 1Password was not present on the list. So while adding it, I modified it below as well by adding a tag. It wasn't necessary but added to make the complete thing.

Only its inclusion in the sponsors section was necessary.

Comment (Abinashbunty) [reply README.md]: I removed the tag now. So only its inclusion in the list was the change. :smile:

Comment (Abinashbunty) [reply README.md]: I added the references to development section and also styled the single line codes in "How to use" section.

**Comment by Abinashbunty** [comment]
I added the references to development section and also styled the single line codes in "How to use" section.

**Comment by Abinashbunty** [comment]
Hi @AtsushiSakai Can this be reviewed? Thanks! 😄

**Comment by AtsushiSakai** [comment]
i’m so sorry, I’m very busy in this month.. Let me check later.

**Comment by Abinashbunty** [comment]
Sure. No worries. Just thought of letting you know that I had made the changes you had requested. 😃😅

**Review by AtsushiSakai — APPROVED** [review]
LGTM. Thank you.
